scripts/basic-workflow-tests.sh

```diff
@@ -179,29 +179,30 @@ test_list() {
 test_serve() {
     MODEL_TYPE="$1"
     SERVE_ARGS=()
+    local MODEL_PATH
```

Comment:
Would the local be better in lowercase?

Reply:
+1 since it's not a global definition

Reply:
I pushed up a commit showing what that would look like - can remove it if desired

Reply:
> Not opposed but this would be inconsistent with how serve, evaluate and phased training does it currently - I think if I change it here I should change it everywhere - wdyt?

+1



> I pushed up a commit showing what that would look like

Consistency is probably most important.  Any reason not to change the other locals if we are going to change model_path?

Reply:
Like do the rest of the locals in this PR? I can for sure, just need to take a closer look at the script

Reply:
I wonder if we can also enforce this with shellcheck somehow?